src/js/modules/components/modal.js

```diff
@@ -0,0 +1,49 @@
+import { View } from 'backbone';
+import modalTemplate from './modal.hbs';
+
+const HIDDEN_CLASS = 'hidden';
+const CONTENT_CLASS = 'plantingjs-modal-content';
+
+export default View.extend({
+  events: {
+    'click .plantingjs-modal-background': 'hide',
+    'click .plantingjs-modal-close-btn': 'hide',
+  },
+
+  constructor({ childView, ...opts }) {
+    this.childView = childView;
+    View.call(this, opts);
```

Comment:
No, is the way to call "super" in backbone. I'm extracting childView in constructor because I want to prevent from any accidental references.